x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/action/TransportExplainLifecycleAction.java

```diff
@@ -152,7 +152,7 @@ static IndexLifecycleExplainResponse getIndexLifecycleExplainResponse(
                     originationDate != -1L ? originationDate : lifecycleState.lifecycleDate(),
                     lifecycleState.phase(),
                     lifecycleState.action(),
-                    lifecycleState.step(),
+                    indexLifecycleService.policyExists(policyName) == false ? ErrorStep.NAME : lifecycleState.step(),
```

Comment:
Can you add a comment above this line explaining why we ingest the "error" name here?